source/resource/qx/tool/schema/compile-1-0-0.json

```diff
@@ -299,6 +299,10 @@
             "type": "object",
             "description": "Options given to @babel/preset-env. With this options the output type of babel can be defined. For details see here: <https://babeljs.io/docs/en/babel-preset-env#options>. They can be overridden per target."
           },
+          "qooxdooVersion": {
+            "type": "string",
+            "description": "The semver that must be satisifed when selecting the Qooxdoo version to compile against; compiler will download a version automatically if the default Qooxdoo does not satisfy this semver"
+          },
```

Comment:
It seems wrong to have a file name with 3 levels, including minor revision, but then to be making changes to the schema without updating the file name. Can we either update the filename to `compile-1-0-1`, or change the filename to one that doesn't include the extra 0's, i.e., `compile-1.json` if we plan to make changes to it without updating its name?

Reply:
that's the problem we've had with updating the manifest schema, it makes all existing .json files incompatible, because the compiler does not have the ability to consider multiple, compatible, schemas.  Modifying the 1.0.0 is just a practical solution

Reply:
Can we simply rename it to `compile-1.json` so it's clear that the filename isn't modified with each change? It's not a big deal to me, but it seems like it would be more clear. There would then be a single incompatibility with this change, and then no further ones.

Reply:
the problem is that all the existing `compile.json` refer to the schema as the `compile-1-0-0`, so if we rename it then they will all fail validation.

I suspect that the solution would be to code a simple transform from 1.0.0 to 1.0.1 to 1.0.2 etc, so that teh compiler is coded to use the latest, but while loading the compile .json will upgrade it if at all possible